test/data/users.js

```diff
@@ -5,8 +5,8 @@ module.exports = {
     email: 'william.salazar@wolox.co',
     password: 'thisIsAVerySecurePassword'
   },
-  NotValidNameMessage: 'Not valid name',
-  NotValidPasswordMessage: 'Not valid password',
+  NotValidNameMessage: 'Name cannot be null or empty',
+  NotValidPasswordMessage: 'Password length should be minimum 8',
```

Comment:
These error messages should be imported from the same constant of the validation middleware.

Reply:
done